package.json

```diff
@@ -4,9 +4,11 @@
   "description": "Remote new tab (third version)",
   "main": "src/main.js",
   "scripts": {
-    "start": "npm run clean && npm run watch:js & npm run watch:css",
-    "build": "npm run clean && npm run build:js && npm run build:css && npm run build:offline",
+    "setup": "npm run clean && npm run build:static",
+    "start": "npm run setup && npm run watch:js & npm run watch:css",
+    "build": "npm run setup && npm run build:js && npm run build:css && npm run build:offline",
     "clean": "rm -rf ./www",
+    "build:static": "cp -r src/static/* www",
```

Comment:
And I think that's because I didn't have a www/ directory yet. Need to make sure it works with or without the www/ directory existing I guess.